doctr/__main__.py

```diff
@@ -135,6 +134,8 @@ def get_parser(config=None):
     deploy_parser_add_argument('--built-docs', default=None,
         help="""Location of the built html documentation to be deployed to
         gh-pages. If not specified, Doctr will try to automatically detect build location""")
+    deploy_parser.add_argument('--deploy-branch-name', default='gh-pages',
+                               help="""Name of branch to deploy to""")
```

Comment:
I guess make this `default=None, help="""Name of the branch to deploy to (default: 'master' for *.github.io repos, 'gh-pages' otherwise"""`